src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -687,6 +687,12 @@ public static void fireChunkWatch(boolean watch, ServerPlayerEntity entity, Chun
             MinecraftForge.EVENT_BUS.post(new ChunkWatchEvent.UnWatch(entity, chunkpos, world));
     }
 
+    public static void fireChunkWatch(boolean wasLoaded, boolean load, ServerPlayerEntity entity, ChunkPos chunkpos, ServerWorld world)
+    {
```

Comment:
Why does this one line helper function exist?

Reply:
Partly to reduce patch size, unless you accept formatting like
```
if (x) foo();
```
in patches. The other part is "historic", in the initial version of the 1.15 PR I replaced the old helper function by this one, but then had to add the old one back to keep binary compatibility.